nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -405,7 +405,14 @@ void CollisionMonitor::process(const Velocity & cmd_vel_in, const std_msgs::msg:
   }
 
   // Points array collected from different data sources in a robot base frame
-  std::vector<Point> collision_points;
```

Comment:
This variable is no longer defined but appears to still be used. Does this compile / work?